recipes/gapseq/build.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Installation instructions taken from https://github.com/jotech/gapseq/blob/master/docs/install.md#conda April 2024 by cmkobel
+
+# Copy contents to conda prefix
+mkdir -p ${PREFIX}/gapseq/
+cp ISSUE_TEMPLATE.MD LICENSE README.md gapseq gapseq_env.yml ${PREFIX}/gapseq/
+cp -r dat/ docs/ src/ toy/ unit/ ${PREFIX}/gapseq/
+
+# Install one additional R-package
+# CHNOSZ is now available on bioconda and is already part of the dependencies.
+# R -e 'install.packages("CHNOSZ", repos="http://cran.us.r-project.org")' 
```

Comment:
We should not install R packages this way.
I guess we need to go the extra round and create those packages in conda-forge. https://conda-forge.org/docs/maintainer/adding_pkgs/#generating-the-recipe

I can help you reviewing them over at conda-forge if you ping me.

Reply:
This is just a comment from the "conda" installation instructions of the gapseq package that I'm basing this recipe off of. The package is installed through conda r::chnosz.

Reply:
Yes, see below, we need everything in conda-forge or bioconda.

Reply:
I submitted a recipe to conda-forge using your skeleton helper.
https://github.com/conda-forge/staged-recipes/pull/26190

I believe that it is ready for merging.